webview/src/experiments/components/App.test.tsx

```diff
@@ -25,7 +26,7 @@ import { vsCodeApi } from '../../shared/api'
 import {
   commonColumnFields,
   expectHeaders,
-  tableData as sortingTableDataFixture
+  tableData as simplifiedSortedTableDataFixture
```

Comment:
Renamed this to differentiate from the `sortedTableDataFixture` which has data based on the `expShow` data. I did consider replacing `simplifiedSortedTableData` with the backend one, but that would complicate multiple tests so I decided to keep it.